subsys/bluetooth/controller/ll_sw/lll_conn_iso.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2021 Demant
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+struct node_tx_iso {
+	union {
+		void        *next;
+		memq_link_t *link;
+	};
+
+	uint64_t payload_number : 39; /* cisPayloadNumber */
+	uint8_t  pdu[];
+};
+
+struct lll_conn_iso_strm_rxtx {
+	uint8_t phy;            /* PHY */
+	uint8_t burst_number;   /* Burst number (BN) */
+	uint8_t flush_timeout;  /* Flush timeout (FT) */
+	uint8_t max_octets;     /* Maximum PDU size */
+};
+
+struct lll_conn_iso_strm {
+	/* Link to ACL connection (for encryption, channel map, crc init) */
+	struct lll_conn *conn;
+
+	/* Connection parameters */
+	uint8_t  access_addr[4];    /* Access address */
```

Comment:
Just out of curiosity: Why is a this a 4 octet array instead of an uint32_t?

Reply:
Because then it looks the same on little-endian and big-endian machines (and the endianness is not easily changed by accident), which helps a lot when comparing big-endian/little-endian traces.